torch/_dynamo/variables/torch.py

```diff
@@ -590,6 +590,14 @@ def check_any_unspec(x):
                 ),
                 **options,
             )
+            is_inplace_func = False
+            try:
+                if fn_.__name__[-1] == "_" or fn_._name[-1] == "_":
```

Comment:
why do we need a try catch here ?

Reply:
For fn_ that is an instance of torch._ops.OpOverload, fn_.__name__ looks like "xxx.default". To get its opname, we use fn_._name here, but this function is not an attribute of ''builtin_function_or_method' object.

Reply:
So just check the existence of `_name` attribute before checking its value would be good enough?